packages/terra-button/src/Button.jsx

```diff
@@ -233,6 +233,11 @@ class Button extends React.Component {
       </span>
     );
 
+    let ariaLabel = customProps['aria-label'];
+    if (isIconOnly || variant === 'utility') {
```

Comment:
nit: I know this was moved from the lines below..we could use `ButtonVariants.UTILITY` instead of 'utility'.